### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by Sosaristic**
## Description

Please include a summary of the change and which issue is fixed (if applicable).

## Related Issue

- Fixes # (issue number)

## Contribution Guidelines

Before submitting this pull request, please review our [Contribution Guidelines](https://github.com/abeg-help/frontend/blob/dev/CONTRIBUTING.md) to understand how to contribute to this project.

## Checklist

- [ ] I have reviewed the Contribution Guidelines linked above.
- [ ] I have tested my changes thoroughly and ensured that all existing tests pass.
- [ ] I have provided clear and concise commit messages.
- [ ] I have updated the project's documentation as necessary.

### CODE REVIEW

**Review by 1finedev — CHANGES_REQUESTED** [review]
Do not re create the design system elements yourself, check in the /components/primitives folder for the button element and others generated to be used on this project

File: pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const NotFound = () => {
+  return (
+    <main className=" min-h-screen flex px-4 flex-col-reverse justify-center bg-abeg-neutral-60 lg:flex-row">
+      <section className="relative flex-1 flex flex-col gap-4 items-center justify-start lg:justify-center lg:w-1/2">
+        <p className="text-2xl font-bold">Oooops!</p>
+        <p className="text-lg font-semibold text-center">
+          We can&apos;t seem to find the page you are looking for
+        </p>
+        <Link href="/">
+          <button className="bg-abeg-green h-[3rem] focus:outline-none focus:ring  focus:ring-abeg-neutral-70 px-4 py-[0.875rem]  flex items-center justify-center rounded-[0.5rem] text-abeg-neutral-70">
```
Comment (289Volts) [inline pages/404.tsx]: is there any need for a button here when you are wrapping it with a link component?

File: pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const NotFound = () => {
+  return (
+    <main className=" min-h-screen flex px-4 flex-col-reverse justify-center bg-abeg-neutral-60 lg:flex-row">
```
Comment (289Volts) [inline pages/404.tsx]: On mobile and tablet screens, the content of the page should be centered on the screen. It is currently starting at the top of the screen.

File: pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          We can&apos;t seem to find the page you are looking for
+        </p>
+        <Link href="/">
+          <button className="bg-abeg-green h-[3rem] focus:outline-none focus:ring  focus:ring-abeg-neutral-70 px-4 py-[0.875rem]  flex items-center justify-center rounded-[0.5rem] text-abeg-neutral-70">
+            Go Home
+          </button>
+        </Link>
+      </section>
+      <section className="relative flex-1 lg:w-1/2">
+        <Image
```
Comment (289Volts) [inline pages/404.tsx:20]: Your image is looking squished when you get to large screens and it looks short on medium to small screens. Please check your sizing well and use devtools to preview

File: pages/404.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          src="/404.webp"
+          alt="404 page image illustration"
```
Comment (289Volts) [inline pages/404.tsx]: There is no need for alt text as the image doesn't convey any meaning. It also has the text on the screen to convey the information

**Review by 289Volts — COMMENTED** [review]
Good job so far. Please see to the fixes and we'll be good to go

File: pages/404.tsx (hunk 4 above)
Comment (1finedev) [reply pages/404.tsx]: i do not think next image will allow images without alt text

Comment (289Volts) [reply pages/404.tsx]: It will. It just needs to be empty. That was what i meant

File: pages/404.tsx (hunk 3 above)
Comment (Sosaristic) [reply pages/404.tsx:20]: i forget say u dey use tv. let me fix all issues.

File: pages/404.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+const NotFound = () => {
+  return (
+    <main className=" min-h-screen flex px-4 flex-col-reverse justify-center items-center bg-abeg-neutral-60 lg:flex-row">
+      <section className="relative  flex flex-col gap-4 items-center justify-start lg:justify-center lg:w-1/2">
+        <p className="text-2xl font-bold">Oooops!</p>
+        <p className="text-lg font-semibold text-center">
+          We can&apos;t seem to find the page you are looking for
+        </p>
+        <Link href="/">
+          <Button className="" variant="primary">
```
Comment (289Volts) [inline pages/404.tsx]: remove this button here

**Review by 289Volts — CHANGES_REQUESTED** [review]
Fix pending issues

**Review by 289Volts — APPROVED** [review]
Good job. Thank you for your contribution

**Pull request merged**